contrib/init/systemd/docker.service

```
@@ -10,7 +10,8 @@ Type=notify
 # the default is not to use systemd for cgroups because the delegate issues still
 # exists and systemd currently does not support the cgroup feature set required
 # for containers run by docker
-ExecStart=/usr/bin/dockerd -H fd://
+# explicitly set the PID file path to be aligned with PIDFile option in this file 
+ExecStart=/usr/bin/dockerd -H fd:// --pidfile /var/run/docker.pid
```

We should probably also add the containerd socket here; like in https://github.com/docker/docker-ce-packaging/blob/master/systemd/docker.service#L14

```suggestion
ExecStart=/usr/bin/dockerd -H fd:// --containerd=/run/containerd/containerd.sock --pidfile /var/run/docker.pid
```

(see https://github.com/docker/docker-ce-packaging/pull/297 for details on that)